app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,121 @@
+<form class="ui form" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui yellow inverted segment">
+      <div class="ui tablet stackable grid">
```

Comment:
<img width="662" alt="screen shot 2017-06-07 at 11 15 27 pm" src="https://user-images.githubusercontent.com/2404372/26892852-3da12798-4bd7-11e7-9187-afb54930721d.png">

The result.